src/components/ui/InputText/input-text.scss

```diff
@@ -72,7 +72,7 @@
 
 [data-fs-input-text-error="true"] {
   input {
-    border-color: var(--fs-color-danger-text);
+    border-color: var(--fs-color-danger-border);
```

Comment:
Hmm, at first glance I was thinking of using `--fs-border-color-danger` instead. But afterward, I notice that we have `Situations tokens` and it LGTM!